Extension/src/common.ts

```diff
@@ -613,13 +644,19 @@ export function spawnChildProcess(process: string, args: string[], workingDirect
     return new Promise<void>(function (resolve, reject): void {
         const child: child_process.ChildProcess = child_process.spawn(process, args, { cwd: workingDirectory });
 
-        child.stdout.on('data', (data) => {
-            dataCallback(`${data}`);
-        });
+        let stdout: Readable | null = child.stdout;
```

Comment:
It would be simpler to use `child.stdout?.on(...)`

Reply:
That trips lint for [no-unused-expressions](https://eslint.org/docs/rules/no-unused-expressions), for some reason.